### PR TITLE
add delete index

### LINKED ISSUE #5: Delete Index

### PR DESCRIPTION
**Pull request by Tumao727**
fix #5 

delete

### CODE REVIEW

File: client/src/i18n/cn/index.ts [hunk 1]
```diff
@@ -7,6 +7,8 @@ const indexTrans = {
   metric: 'Metric Type',
 
   createSuccess: 'Creating index successfully',
+  deleteWarning:
+    'You are trying to delete a index. This action cannot be undone.',
```
Comment (shanghaikid) [inline client/src/i18n/cn/index.ts:11]: delete an index

File: client/src/i18n/en/index.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ const indexTrans = {
 
   metric: 'Metric Type',
   createSuccess: 'Creating index successfully',
+  deleteWarning:
+    'You are trying to delete a index. This action cannot be undone.',
```
Comment (shanghaikid) [inline client/src/i18n/en/index.ts:11]: delete an index

**Review by shanghaikid — CHANGES_REQUESTED** [review]

**Review by nameczz — APPROVED** [review]

**Comment by Tumao727** [comment]
delete

**Pull request merged**